libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -269,7 +271,10 @@ bool ASTJsonConverter::visit(ContractDefinition const& _node)
 		make_pair("contractKind", contractKind(_node.contractKind())),
 		make_pair("abstract", _node.abstract()),
 		make_pair("baseContracts", toJson(_node.baseContracts())),
-		make_pair("contractDependencies", getContainerIds(_node.annotation().contractDependencies, true)),
+		make_pair("contractDependencies", getContainerIds(
+			_node.annotation().contractDependencies | views::keys,
+			true
```

Comment:
Aren't they already sorted by ID?

Reply:
The change here is not due to sorting but due to the type change to a map of `contractDependencies`

Reply:
I know, but you can remove the `true` now, can't you?

Reply:
Ah, right. I didn't see that the function also does ordering (not obvious from the name)